.goreleaser.yml

```diff
@@ -0,0 +1,49 @@
+env:
+- GOPROXY=https://proxy.golang.org|direct
+- GO111MODULE=on
+- CGO_ENABLED=0
+- PKG=github.com/operator-framework/rukpak/internal/version
+- IMAGE_REPO=quay.io/operator-framework/plain-v0-provisioner
+before:
+  hooks:
+    - go mod tidy
+    - go mod download
+builds:
+  - id: unpack
+    main: ./cmd/unpack
+    binary: unpack
+    goos:
+    - linux
+    goarch:
+    - amd64
+    ldflags:
+    - -X {{ .Env.PKG }}.GitCommit={{ .ShortCommit }}
+  - id: plain-v0
+    main: ./provisioner/plain-v0
+    binary: plain-v0
+    goos:
+    - linux
+    goarch:
+    - amd64
```

Comment:
note: linux-amd64 is the only supported container image right now. I just wanted to get consensus on this before I expand the list of archs (follow-up?).

Reply:
I'd say a follow-up would make sense as we're already adding a lot here.

Reply:
I think I'll look into combing the deploy + release workflow in a follow-up PRs as well, so both of them build and publish containers through goreleaser.

Reply:
Yeah that sounds like a great idea.